pkg/replica/diff_disk.go

```diff
@@ -372,12 +372,17 @@ func (d *diffDisk) initializeSectorLocation(value byte) {
 	}
 }
 
-func (d *diffDisk) preload() error {
+func (d *diffDisk) preload(isThereBackingFile bool) error {
 	for i, f := range d.files {
 		if i == 0 {
 			continue
 		}
 
+		// Skip loading the backingFile if it exist
+		if i == 1 && isThereBackingFile {
```

Comment:
I think we can use the const `int(backingFileIndex)` here so we won't confuse what does "1" mean